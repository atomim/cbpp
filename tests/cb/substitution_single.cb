#define RADTODEG(a) ((a/(2.0*PI))*360.0)

rads# = 1.0
AddText rads + " radians = " + RADTODEG(rads) + " degrees "
DrawScreen
WaitKey