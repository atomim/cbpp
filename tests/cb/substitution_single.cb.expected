// #define RADTODEG(a) ((a/(2.0*PI))*360.0)

rads# = 1.0
AddText rads + " radians = " + ((rads/(2.0*PI))*360.0) + " degrees "
DrawScreen
WaitKey